test/e2e/rukpakctl_test.go

```diff
@@ -232,16 +233,31 @@ var _ = Describe("rukpakctl content subcommand", func() {
 	When("content executed with a wrong bundle name", func() {
 		var (
 			output string
+			err    error
 		)
 		BeforeEach(func() {
-			out, err := exec.Command("sh", "-c", rukpakctlcmd+"content badname").Output()
-			Expect(err).NotTo(BeNil())
+			var out []byte
+			out, err = exec.Command("sh", "-c", rukpakctlcmd+"content badname").Output()
+			Expect(err).To(HaveOccurred())
 			output = string(out)
 		})
 
-		It("output error message", func() {
-			Expect(strings.Contains(output, "content command failed") &&
-				strings.Contains(output, "bundles.core.rukpak.io \"a\" not found"))
```

Comment:
`output` (stdout) was empty becasue rukpak logs errors into stderr. So boolean expression was evaluating into `false`. Since there was no matcher - this was testing nothing.

Good linter 🐶